tox.ini

```diff
@@ -73,6 +73,7 @@ deps =
   -c dev-requirements.txt
   opentelemetry: pytest
   opentelemetry: pytest-benchmark
+  opentelemetry: pytest-randomly
```

Comment:
Just to confirm, just having `pytest-randomly` installed will run any subsequent `pytest` call execute tests in a random order correct? Also, I am assuming that the only tests that needed to be fixed due to order are the ones above?

Reply:
Correct - `pytest-randomly` is enabled by default, using seed from `random` module (`random.seed()`), which is based on unixtime. There are a couple more tests needed to fix, I'll utilize @srikanthccv helpful comments and implement the fixes in a sub-base class inheriting from `BaseClass`